docs/standard/simd.md

```diff
@@ -0,0 +1,103 @@
+---
+title: SIMD in .NET
+description: This tutorial demonstrates how to use hardware SIMD operations in C# and .NET Core.
+author: FIVIL
+ms.author: tagoo
+ms.date: 03/31/2020
+ms.technology: dotnet-standard
+---
+
+# Overview
+
+SIMD (Single instruction, multiple data) provides hardware support for performing an operation on multiple pieces of data, in parallel, using a single instruction. In .NET, there's set of SIMD-accelerated types under the <xref:System.Numerics> namespace. SIMD operations can be parallelized at the hardware level. That increases the throughput of the vectorized computations, which are common in mathematical, scientific, and graphics apps.
+
+## .NET SIMD-accelerated types
+
+The .NET SIMD-accelerated types include the following types:
+
+- The <xref:System.Numerics.Vector2>, <xref:System.Numerics.Vector3>, and <xref:System.Numerics.Vector4> types, which represent vectors with 2, 3, and 4 <xref:System.Single> values.
+
+- Two matrix types, <xref:System.Numerics.Matrix3x2>, which represents a 3x2 matrix, and <xref:System.Numerics.Matrix4x4>, which represents a 4x4 matrix.
```

Comment:
Should we call out that this is xref:System.Single values, like we did for Vector2/3/4?

Likewise for Plane/Quaternion?

Reply:
Would you be willing to write that content? It's a bit out of my area of expertise.

Reply:
```suggestion
- Two matrix types, <xref:System.Numerics.Matrix3x2>, which represents a 3x2 matrix, and <xref:System.Numerics.Matrix4x4>, which represents a 4x4 matrix of <xref:System.Single> values.
```